packages/apps/demo/src/components/TopNav.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright 2023 Comcast Cable Communications Management, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Base, context, Row, TextBox } from '@lightningjs/ui-components';
+
+export default class TopNav extends Base {
+  static get __componentName() {
+    return 'TopNav';
+  }
+
+  static _template() {
+    return {
+      w: (app.stage.w / app.stage.getRenderPrecision()) - context.theme.layout.safe * 2,
+      y: context.theme.layout.safe,
+      x: context.theme.layout.safe,
+      Logo: {
+        w: 98,
+        h: 48
+      },
+      Controls: {
+        x: w => w,
+        mountX: 1,
+        autoResizeWidth: true,
+        type: Row,
+        items: [
+          {
+            type: class extends TextBox {
```

Comment:
can we break this out, even if it stays in this file? Or if the goal is for this to be a clock, just make a clock component?